pydantic/fields.py

```diff
@@ -340,13 +340,35 @@ def __repr_args__(self) -> ReprArgs:
                     yield s, value
 
 
-def Field(
+class AliasPath:
+    def __init__(self, first_arg: str, *args: str | int) -> None:
+        self.paths = [first_arg] + list(args)
+
+    def convert_to_aliases(self) -> list[str | int]:
+        return self.paths
```

Comment:
Looking farther I see why you did this, it lets you call the same method below. That's reasonable to me. Ignore the above (though I still think we should rename to `path` rather than `paths` and use `@slots_dataclass`).

Reply:
Yes, exactly.
Renaming `paths` to `path` and using `@slots_dataclass` make sense. I will update